integration-tests/smallrye-opentracing/pom.xml

```diff
@@ -43,15 +43,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-agroal</artifactId>
         </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy-mutiny</artifactId>
-        </dependency>
 
         <!-- Client dependencies -->
         <dependency>
```

Comment:
I'm thinking about this and I'm not sure it's worth making this test a Frankenstein. I think I would just drop the testing on this one.

But YMMV.

Reply:
I actually wanted to have this test because I think it's relatively easy for us to miss this in the future